README.pod

```diff
@@ -268,6 +269,15 @@ in a file named `Build.pm` at the root of your distribution:
         }
     }
 
+NOTE: You may use installed modules inside the build hook, but those modules
+must be listed in the `META6.json` file in either the "build-depends" or "depends"
+section, as appropiate. You also may use the module being built by adding
+something similar to the following in the build hook file:
+
+    use lib <./lib>; # ensures the local version is being used
```

Comment:
This is bad. It should be assumed one cannot use anything in `lib` until the build phase has been completed (even if its true that it isn't strictly required many times). It also assumes that the build command is being run in a specific directory, but technically I should be able to run it from other directories.

Reply:
Okay, then I'll take out the sentence that describes the local lib use as well as that example.

Is it okay to use a code fragment calling a function via a direct file reference somehow? Hm, but I guess that wouldn't work either.

Is there any way to, via the build hook, unpack the distro into a temp directory for testing? And then, if successful, skip on to the install phase?

The use case I'm working on, Documentable, IMO, needs to modify the directory it is testing, and then clean it up. Some can be done via t/*t files, but that is cumbersome, too. I was hoping the build hook could help with that, but it will be harder without duplicating some Documentable code.

Reply:
> Is there any way to, via the build hook, unpack the distro into a temp directory for testing? And then, if successful, skip on to the install phase?

That would be something a post-test hook could do, although such a thing does not exist yet. It actually existed as a package-manager-agnostic implemention [4-5 years ago](https://github.com/ugexe/zef/tree/36d643542d52c20f50f1261ad4aae178d4e07de0/hooks) but I removed it during a refactor because no one was using it (or zef really) at the time.